abuse/signup.md

```diff
@@ -123,6 +124,37 @@ rollover procedures and secure key storage.
 [TextCaptcha]: http://textcaptcha.com/
 
 
+#### Key management considerations
+
+Since no persistent key is used, most of the usual key management and
+distribution woes are side-stepped.  However, we must be careful that
+the transient encryption key is not kept too long.
+
+The theoretical limit on how many tokens can be safely encrypted with a 128-bit
+cipher is 2⁶⁴ bits, and Go standard nonce size is 96 bits, leading to 2⁴⁸
+messages with random nonces before a nonce reuse can be not unlikely.
+
+Both are comfortably over what can be expected to be served by the API server
+over its lifetime (i.e. without restart).  However, less-than-perfect random
+number generation can significantly increase the risk of accidental nonce reuse.
+
+More importantly, a variety of other factors can lead to key compromise:
+vulnerabilities in our own code, flawed crypto implementation, ...
+
+To mitigate this, the implementation should enforce that the encryption key is
+volatile, by replacing it with a new random one, whenever a new `/captcha`
+request occurs and either of those conditions are met:
```

Comment:
Both seem legit.
No strong feeling about it, I can break out the *Chicago Manual of Style* when I'm back...  :P